lib/sms.js

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+var Nexmo = require('nexmo')
+var P = require('bluebird')
+
+var TEMPLATE_NAMES = new Map([
+  [ 1, 'installFirefox' ]
+])
+
+module.exports = function (log, translator, templates, smsConfig) {
+  var nexmo = new Nexmo({
+    apiKey: smsConfig.apiKey,
+    apiSecret: smsConfig.apiSecret
+  })
+  var sendSms = P.promisify(nexmo.message.sendSms, { context: nexmo.message })
+  var installFirefoxLink = smsConfig.firefoxAppStoreLink
```

Comment:
If the name is just a generic `link`, what happens in the future if we add other message types that contain different links? In that scenario, we want the specific name so that we can distinguish between them in the code and have some kind of map from message ids to link names, like there is for message ids to template names already.

Actually, following that train of thought further, it strikes me that the link name should be the same as the template name, if they are part of a pair. So both called `installFirefox` in this case, and both keyed by the message id. What do you think?